lib/ansible/modules/cloud/vmware/vmware_serial_port.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Anusha Hegde <anushah@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vmware_serial_port
+
+short_description: Create a serial port
+
+version_added: "1.0"
+
+description:
+    - "Creates a serial port on an existing VM"
+
+options:
+    hostname:
```

Comment:
Please use `extends_documentation_fragment: vmware.documentation` instead of specifying `hostname`, `username` and `password`